Sleep Tracker/Sleep Tracker/View/DuringSleepView/DuringSleepView.swift

```diff
@@ -0,0 +1,23 @@
+import SwiftUI
+
+struct DuringSleepView: View {
+
+    // MARK: - Public interface
+    var body: some View {
+        VStack {
+            Image("Sleeping_astronaut_image")
```

Comment:
I'd like to have typed image names. Smth like:

```suggestion
            Image(.astronautImage)
```

Otherwise, it's easy to have a typo plus we follow the DRY principle.

Reply:
fixed